src/components/RankingTableRow.js

```diff
@@ -2,33 +2,37 @@ import React from 'react'
 import injectSheet from 'react-jss'
 import clsx from 'clsx'
 
-import { black } from 'constants/styles/colors'
+import { black, yellow } from 'constants/styles/colors'
+import { robotoBold } from 'constants/styles/fonts'
 
 const styles = {
 	row: {
 		display: 'flex',
-		borderBottom: [[1, 'solid', black]],
+		marginBottom: 5,
 		width: '100%',
-		'&:last-child': {
-			borderBottom: 'none',
-		},
 	},
 	rankBox: {
 		padding: 4,
-		textAlign: 'center',
 		width: '15%',
-		borderRight: [[1, 'solid', black]],
+		border: [[2, 'solid', black]],
```

Comment:
Just as an FYI, this notation and `border: '2, solid, black'` are the same. It doesn't make a difference which one you use.